code/game/objects/items/attachments/shoulder_sling.dm

```diff
@@ -0,0 +1,31 @@
+/obj/item/attachment/sling
+	name = "shoulder sling"
+	desc = "A leather shoulder sling for longarms to allow for easy carrying on the shoulder without the need for traditional armor holsters."
+	icon_state = "sling"
```

Comment:
You didn't rename the icon state in the DMI

Reply:
Fixed